### PR TITLE
Initial schema design and create sample data

### PR DESCRIPTION
**Pull request by kbie**
notes for this initial schema design
- no hierarchy in location and catogory for now
- no time matching for now
- no required fields, this will be managed by app

### CODE REVIEW

File: backend/models/location.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+var mongoose = require('mongoose');
+
+var locationSchema = {
```
Comment (mxiasj) [inline backend/models/location.js]: This location looks a bit confusing. It seems to be the city-level location (e.g., SV, LA, Hou) not the service-requesting location.

File: backend/models/order.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+var mongoose = require('mongoose');
+var Category = require('./category');
+
+var orderSchema = {
```
Comment (mxiasj) [inline backend/models/order.js]: Should we have an order id field?

File: backend/models/user.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+var mongoose = require('mongoose');
+
+var userSchema = {
+  profile: {
+    username: { type: String, lowercase: true },
+    name: { type: String },
```
Comment (mxiasj) [inline backend/models/user.js]: first name & last name?

**Comment by changran52m** [comment]
👍

File: backend/models/location.js (hunk 1 above)
Comment (changran52m) [inline backend/models/location.js]: Maybe we can consider to use [GeoJSON](https://github.com/Automattic/mongoose/wiki/3.6-Release-Notes#geojson-support-mongodb--24), when moving forward. The is a standard format to store location, so we don't need to manage `location model` by ourselves.

In short, I would like to remove this file/model. 😃

Comment (kbie) [inline backend/models/location.js]: will remove

File: backend/models/order.js (hunk 2 above)
Comment (kbie) [inline backend/models/order.js]: it is auto generated

File: backend/models/user.js (hunk 3 above)
Comment (kbie) [inline backend/models/user.js]: will add

**Pull request merged**